cpp/ql/src/Security/CWE/CWE-676/PotentiallyDangerousFunction.qhelp

```diff
@@ -48,7 +52,7 @@ rules for the following CWEs:</p>
 
 </section>
 <references>
-
-
+<li>Wikipedia: <a href="http://en.wikipedia.org/wiki/Morris_worm">Morris worm</a>.</li>
+<li>E. Spafford. <i>The Internet Worm Program: An Analysis</i>. Purdue Technical Report CSD-TR-823, <a href="http://www.textfiles.com/100/tr823.txt">(online)</a>, 1988.</li>
```

Comment:
Surely https://cwe.mitre.org/data/definitions/242.html is the canonical source for this query.

Can we find a source that specifically mentions `gmtime`? The best I can find is https://wiki.sei.cmu.edu/confluence/display/c/CON33-C.+Avoid+race+conditions+when+using+library+functions. It also recommends against a bunch of other functions. Maybe now is a good time to extend the `gmtime` rule to also cover `asctime`, `ctime` and `localtime`? See also the discussion at https://semmle.slack.com/archives/C7NS4H8HY/p1542190176111100.

Reply:
> Surely https://cwe.mitre.org/data/definitions/242.html is the canonical source for this query.

I believe that reference is added automatically as long as the tag is present.

> Can we find a source that specifically mentions gmtime? The best I can find is https://wiki.sei.cmu.edu/confluence/display/c/CON33-C.+Avoid+race+conditions+when+using+library+functions.

That looks fine, added.

> Maybe now is a good time to extend the gmtime rule to also cover asctime, ctime and localtime? See also the discussion at https://semmle.slack.com/archives/C7NS4H8HY/p1542190176111100.

Good idea.  I'd rather do that in a follow-up PR as it will involve another round `qhelp` changes and a change note.